content/millennium/r4/clinical/diagnostics/diagnostic-report.md

```diff
@@ -61,12 +66,12 @@ The following fields are returned if valued:
 ------------------|-------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------
```

Comment:
similar to how you included it for by ID, can you include the intro here? example from Patient:

> Search for Patients that meet supplied query parameters:
> 
> `GET /Patient?:parameters`

Reply:
yeah, adding that nowish